@remirror/core-extensions/src/extensions/history-extension.ts

```diff
@@ -108,27 +61,6 @@ export class HistoryExtension extends Extension<HistoryExtensionOptions> {
     return history({ depth, newGroupDelay });
   }
 
-  /**
-   * Provides the isEnabled method to the ActionMethods of undo and redo.
-   *
-   * @remarks
-   *
-   * - Redo is not enabled when at the end of the history and there is nothing left to redo.
-   * - Undo is not enabled when at the beginning of the history and there is nothing left to undo.
-   */
-  public isEnabled({ getState }: ExtensionManagerParams) {
-    return ({ command }: { command?: string }) => {
-      switch (command) {
-        case 'undo':
-          return undoDepth(getState()) > 0;
-        case 'redo':
-          return redoDepth(getState()) > 0;
-        default:
-          return false;
-      }
-    };
-  }
-
```

Comment:
This is not needed because redo() and undo() from prosemirror effectively do these checks when called with dispatch=false